mm2src/mm2_main/src/lp_swap/maker_swap_v2.rs

```diff
@@ -118,6 +118,14 @@ pub enum MakerSwapEvent {
         maker_payment: TransactionIdentifier,
         taker_payment: TransactionIdentifier,
     },
+    /// Taker payment has been received and taker payment spend preimage validation skipped.
+    TakerPaymentReceivedAndPreimageValidationSkipped {
```

Comment:
same here

Reply:
same as above, the whole preimage name is too long to be used in enum, so it is clarified in doc comment

Reply:
updated https://github.com/KomodoPlatform/komodo-defi-framework/pull/2359/commits/2bc305b3d379fbc7ed4674c61b3158a249457e2b